### PR TITLE
Add AD and SF Studios

### PR DESCRIPTION
**Pull request by japierce1028**

### CODE REVIEW

File: src/main/data/studios.json [hunk 1]
```diff
@@ -11,6 +11,10 @@
     "code": "9LE",
     "description": "9 LIGHT ENTERTAINMENT"
   },
+  {
+    "code": "AD",
+    "description": "ART DOYDU"
+  },
```
Comment (SteveLLamb) [inline src/main/data/studios.json:17]: You can now add in the URL and contact info. Use this as an example:

{
    "code": "1VP",
    "contact": {
      "address": "Feiteng Art Space Zoom A, No.173,Caochangdi, Chaoyang district, Beijing, China",
      "email": "mali@123vp.com",
      "name": "Mr.Ma"
    },
    "description": "123 Vision Pictures",
    "url": "https://www.123vp.com"
  },

File: src/main/data/studios.json [hunk 2]
```diff
@@ -794,6 +798,10 @@
     "code": "SEV",
     "description": "Seville International"
   },
+  {
+    "code": "SF",
+    "description": "SakhaFilm"
```
Comment (SteveLLamb) [inline src/main/data/studios.json:803]: Same as above, they are not opted out, all info can be added.

**Review by SteveLLamb — CHANGES_REQUESTED** [review]

**Review by japierce1028 — COMMENTED** [review]
Fixed code per Steve's suggestion.

File: src/main/data/studios.json [hunk 3]
```diff
@@ -11,6 +11,16 @@
...
   },
+  {
+    "code": "AD",
+    "contact": {
+    "address": "57-16, Stroda str., Yakutsk, Sakha Republic, Russia",
+    "email": "bertjickty@yandex.ru",
+    "name": "Mr. Jacob E. Alexandrov"
+    },
+    "description": "ART DOYDU",
+    "url":"http://artdoydu.com"
```
Comment (SteveLLamb) [inline src/main/data/studios.json:22]: Needs a space after colon.

File: src/main/data/studios.json [hunk 4]
```diff
@@ -794,6 +804,16 @@
...
   },
+  {
+    "code": "SF",
+    "contact": {
+    "address": "57-16, Stroda str., Yakutsk, Sakha Republic, Russia",
+    "email": "bertjickty@yandex.ru",
+    "name": "Mr. Jacob E. Alexandrov"
+    },
+    "description": "SakhaFilm",
+    "url":"http://sakhafilm.com/"
```
Comment (SteveLLamb) [inline src/main/data/studios.json:815]: Needs a space after colon.

**Review by SteveLLamb — CHANGES_REQUESTED** [review]

**Review by SteveLLamb — APPROVED** [review]

**Pull request merged**